config/locales/en.yml

```diff
@@ -1,17 +1,118 @@
 ---
 en:
-  availability: &id-availability
-    afternoon: Afternoon
-    detailed_description: Detailed Description
-    evening: Evening
-    flexible: Flexible
-    morning: Morning
-    weekend: Weekend
-    workday: Workday
+  aaa_injectable: &id-injectables
+    assignment_kinds:
+      accompaniment: Accompaniment
+      family: Family
+      workshop: Workshop
+      german_class: German Class
+      transit_center: Transit center
+    age_request: &id-age_requests
+      age_middle: 36 - 50
+      age_no_matter: "age doesn't matter"
+      age_old: older than 51
+      age_young: 20 - 35
+      all: All
+    availability: &id-availability
+      afternoon: Afternoon
+      detailed_description: Detailed Description
+      evening: Evening
+      flexible: Flexible
+      morning: Morning
+      weekend: Weekend
+      workday: Workday
+    category: &id-categories
+      conversation: Conversation
+      email: Email
+      feedback: Feedback
+      file: File
+      telephone: Telephone
+    gender_request: &id-gender_requests
+      all: All
+      no_matter: "gender doesn't matter"
+      same: gender same
+    generic_keys: &id-generic_keys
+      address: Address
+      avatar: Photo
+      birth_year: Birth year
+      city: City
+      created_at: Created at
+      email: Email
+      first_name: First name
+      full_name: Name
+      languages: Languages
+      last_name: Last name
+      nationality: Nationality
+      phone: Phone number
+      postal_code: Zip
+      profession: Profession
+      salutation: Salutation
+      state: State
+      street: Street
+      updated_at: Updated at
+      zip: Postleitzahl
```

Comment:
Is there a reason that this is in German?

Reply:
see above